requests-for-comments/2024-02-13-web-analytics-launch.md

```diff
@@ -0,0 +1,64 @@
+## Web analytics marketing launch plan
+
+The launch date for web analytics isn't nailed down yet, but it's _seeming_ like it'll be in March. Either way, it's never too early to plan for how we'll market the heck out it. 
+
+It's worth noting that Web Analytics has been in opt-in beta for some time too, so we can rely on this group of beta-testing customers as an additional launch asset!
+
+We have some [new best practices for product launches around templates](https://posthog.com/handbook/engineering/development-process#best-practices-for-full-releases), but these mostly do not apply to web analytics due to its scope. 
+
+## Pricing?
+
+Pricing is TBC, but web analytics will have usage-based pricing. 
+
+## Site updates
+
+Basic stuff across the site that'll need to be done. 
+
+- [ ] Update documentation - @andyvan https://github.com/PostHog/posthog.com/issues/7736
+- [ ] Create a product page 
+- [ ] Web analytics case study - @joethreepwood
+- [ ] Remove beta tags from homepage slider, docs, etc - @joethreepwood
+- [ ] Add Web analytics links to footer - @joethreepwood
+- [ ] Update `/blog/ga4-alternatives` with a CTA
+- [ ] Update `/blog/track-your-website-with-posthog` with a CTA
+- [ ] Update `/blog/google-analytics-to-posthog` with a CTA
+- [ ] Update `https://posthog.com/blog/posthog-vs-ga4` with a CTA
+- [ ] Create a Web Analytics section for `/questions`
+- [ ] Add a Frequently Asked Questions post to community - @joethreepwood
+
+## Other updates
+
+Basic stuff outside the website that we'll need to update. 
+
+- [ ] Add Web Analytics as an option in in-app onboarding - GROWTH
+- [ ] Add Web Analytics path as an email onboarding option - @joethreepwood
+- [ ] Add Web Analytics to G2, Bookface, etc - @joethreepwood
+- [ ] Update PostHog README.md with information - @joethreepwood
+- [ ] Add PostHog to any relevant new G2 categories
+
+## Default launch activities
+
+Basic things we always do. 
+
+- [ ] Run a relevant Product for Engineers newsletter: `How we built Web Analytics` / `Learning from the mistakes of GA4` maybe?
+- [ ] In-app announcement using surveys/notifications for all users - @joethreepwood
+- [ ] Email announcement to beta users _ahead of launch_ - @joethreepwood (preferential pricing?)
+- [ ] Email announcement to all remaining users _at launch_ - @joethreepwood
+- [ ] Blog + Changelog announcement at launch - @joethreepwood
+- [ ] Social announcements on Twitter + LinkedIn - @joethreepwood
+- [ ] Pylon announcement to all current paying users - @joethreepwood
+- [ ] New key art for use in blogs, emails, etc - @lottiecoxon
+- [ ] New web analytics ads run by Hey Digital 
+
+## Bigger ideas
+
+Ideas for other, weirder things go here. 
+
```

Comment:
@robbie-c @ivanagas What would you think about adding a tool/doc about how to migrate from GA4 to PostHog? 

I'm unclear on the process here, but I imagine anything we do to support migration of data is going to help encourage adoption.

Reply:
Amplitude and Mixpanel have this in their docs
https://www.docs.developers.amplitude.com/data/tutorials/migrating-from-google-analytics/
https://docs.mixpanel.com/docs/migration/google-analytics

Reply:
Doc yes, tool maybe. Internal thread on GA4 import https://posthog.slack.com/archives/C05LJK1N3CP/p1707910732916289